### DISCUSSION TITLE
Can `AttributeFilter` match against the actual attribute?

### DISCUSSION THREAD

**Discussion by rrrnld** [saleor/saleor#9620] [Q&A]
When writing queries that work against the list of products stored in Saleor, I noticed that the `AttributeFilter` in a `ProductsQuery` matches against the slug form of the attribute's value, not the actual attribute. The query looks like this:

``` graphql
query {
  products(first: 100, channel: "eur", filter: {
    attributes: { slug: "doi", values: ["104171000", "10.4171/000"] }
  }) {
    edges { node { id, name, attributes { attribute { name, slug }, values { name, slug } } } }
  }
}
```

What is matched against in this case is `104171000`, not `10.4171/000`. If I remove the second value the result is unchanged, if I remove the first one it is empty.
 
I found this behavior a bit surprising. Is there a way to match against the attribute's value literally as I stored it?

**Comment by karolkielecki** [comment]
Hi @heyarne, before passing that case to the team, could you please tell me your Saleor version?

**Reply by rrrnld** [reply]
I'm using `3.2.0`

**Reply by karolkielecki** [reply]
Thanks! I've passed it on to the team. Not sure if we can improve it, but we'll discuss it further.

**Reply by rrrnld** [reply]
Are there any news on this? I'm on 3.4.9 now.

**Reply by manojLondhe** [reply]
+2 We want to use this feature too
